servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered.glsl

```diff
@@ -1989,19 +1989,15 @@ void fragment_shader(in SceneData scene_data) {
 		//simplify for toon, as
 		specular_light *= specular * metallic * albedo * 2.0;
 #else
-
-		// scales the specular reflections, needs to be computed before lighting happens,
-		// but after environment, GI, and reflection probes are added
-		// Environment brdf approximation (Lazarov 2013)
-		// see https://www.unrealengine.com/en-US/blog/physically-based-shading-on-mobile
-		const vec4 c0 = vec4(-1.0, -0.0275, -0.572, 0.022);
-		const vec4 c1 = vec4(1.0, 0.0425, 1.04, -0.04);
-		vec4 r = roughness * c0 + c1;
-		float ndotv = clamp(dot(normal, view), 0.0, 1.0);
-		float a004 = min(r.x * r.x, exp2(-9.28 * ndotv)) * r.x + r.y;
-		vec2 env = vec2(-1.04, 1.04) * a004 + r.zw;
-
-		specular_light *= env.x * f0 + env.y * clamp(50.0 * f0.g, metallic, 1.0);
+		// Base Layer
+		float NdotV = clamp(dot(normal, view), 0.0001, 1.0);
+		vec2 envBRDF = prefiltered_dfg(roughness, NdotV).xy;
+		// Multiscattering
+		vec3 energy_compensation = get_energy_compensation(f0, envBRDF.y);
+
+		// cheap luminance approximation
+		float f90 = clamp(50.0 * f0.g, 0.0, 1.0);
+		specular_light *= energy_compensation * (f90 * envBRDF.x + f0 * envBRDF.y);
```

Comment:
I am honestly not sure if we want to also use the dfg on the mobile renderer. I would argue that the current approximation is good enough and rather avoid the additional texture read on mobile. @clayjohn do you have an opinion on that.

Reply:
Yes, I agree mobile should stick with the current approximation. It's best to benchmark this still, but I don't have access to really old/low-end devices currently.

Reply:
I agree. Mobile should use the approximation

Reply:
This code is only on the forward+ renderer. The current changes to the mobile renderer are fine